source/Core/Drivers/BootLogo.cpp

```diff
@@ -36,6 +39,11 @@ void BootLogo::showNewFormat(const uint8_t *ptrLogoArea) {
 
     int len = (showNewFrame(ptrLogoArea + position));
     OLED::refresh();
+    
+    // At end of animation
+    if (len == 0) {
+      goto wait;
```

Comment:
Goto is not a wise idea due to ability to introduce bugs.
I would make the len part of the do{}while loop check instead
i.e. `while( position<1022 && len>0)`

Reply:
Thanks, 😃
didn't think about that.

Will check later.